docs/source/01_intro.md

```diff
@@ -1,5 +1,48 @@
 # Introduction
 
+ADAM is a library and command line tool that enables the use of [Apache
+Spark](https://spark.apache.org) to parallelize genomic data analysis across
+cluster/cloud computing environments. ADAM uses a set of schemas to describe
+genomic sequences, reads, variants/genotypes, and features, and can be used
+with data in legacy genomic file formats such as SAM/BAM/CRAM or VCF, as well
```

Comment:
Yeah, it is down in 55_api.md in the ADAMContext docs. That said, I'm planning to add a graphic in the `GenomicRDD` section that makes the file format<->schema mapping more clear.